cromshell

```diff
@@ -282,7 +282,44 @@ function checkPrerequisites
   return 0
 }
 
-function checkForComplexHelpArgument() 
+function assertRequiredFileIsNonEmpty()
+{
+
+  local fileName=$1
+  local fileDescription=$2
+  if [[ -z ${fileName} ]]
```

Comment:
A couple of style things:

- Can you tabs to spaces this file (where 1 tab = 2 spaces)?  I've been trying to avoid `\t` as much as possible.
- All the if/then clauses should occur on the same line ala `if CONDITION ; then` to keep formatting consistent.

Reply:
ooh, didn't even realize i had tabs there..

Reply:
fixed that